Agents/MetOfficeAgent/metoffice/flaskapp/inputtasks/routes.py

```diff
@@ -0,0 +1,105 @@
+###############################################
+# Authors: Markus Hofmeister (mh807cam.ac.uk) #    
+# Date: 08 Apr 2022                           #
+###############################################
+
+from flask import Blueprint, request, jsonify
+
+#import agentlogging
+from metoffice.datainstantiation.stations import instantiate_all_stations
+from metoffice.datainstantiation.readings import instantiate_all_station_readings
+from metoffice.datainstantiation.readings import add_all_readings_timeseries
+from metoffice.datainstantiation.readings import update_all_stations
+
+
+# Initialise logger
+#logger = agentlogging.get_logger("prod")
+
+inputtasks_bp = Blueprint(
+    'inputtasks_bp', __name__
+)
+
+# Define route for API request to instantiate all stations
+@inputtasks_bp.route('/api/metofficeagent/instantiate/stations', methods=['GET'])
```

Comment:
We need to have a wider discussion about the blanket use of the "GET" method for every route in every agent. For now I think it is fine to stick with it though as it is the current convention in the TWA.